zimui/index.html

```diff
@@ -0,0 +1,13 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <link rel="icon" type="image/x-icon" href="./favicon.ico" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Kolibri channel in a ZIM</title>
```

Comment:
I opened https://github.com/openzim/kolibri/issues/69 because it is not really straightforward, but I strongly agree (and even found a bug around this in content pages, mentioned in the issue)